openedx/features/job_board/models.py

```diff
@@ -15,17 +17,31 @@ class Job(TimeStampedModel):
     """
     title = models.CharField(max_length=255)
     company = models.CharField(max_length=255)
-    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES)
-    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES)
-    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES)
+    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES, blank=False,
+                            default=0, help_text='Please select whether the job is onsite or can be done remotely.')
+    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES,
+                                    blank=False, default=0, help_text='Please select the type of compensation you are '
+                                                                      'offering for this job.')
+    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES,
+                             blank=False, default=0, help_text='Please select the expected number of working hours '
+                                                               'required for this job.')
```

Comment:
shouldn't default should be from choices instead of 0?

Reply:
in the design no values have been selected by default thats why I have put it 0 there.